plugins/woocommerce/src/Blocks/Domain/Services/FeatureGating.php

```diff
@@ -118,54 +61,4 @@ public function is_development_environment() {
 	public function is_production_environment() {
 		return self::PRODUCTION_ENVIRONMENT === $this->environment;
 	}
-
-	/**
-	 * Checks if we're executing the code in a test environment.
-	 *
-	 * @return boolean
-	 */
-	public function is_test_environment() {
-		return self::TEST_ENVIRONMENT === $this->environment;
-	}
-
-	/**
-	 * Returns core flag value.
-	 *
-	 * @return number
-	 */
-	public static function get_core_flag() {
-		return self::CORE_FLAG;
-	}
-
-	/**
-	 * Returns feature plugin flag value.
-	 *
-	 * @return number
-	 */
-	public static function get_feature_plugin_flag() {
-		return self::FEATURE_PLUGIN_FLAG;
-	}
-
-	/**
-	 * Returns experimental flag value.
-	 *
-	 * @return number
-	 */
-	public static function get_experimental_flag() {
-		return self::EXPERIMENTAL_FLAG;
-	}
-
-
-	/**
-	 * Check if the block templates controller refactor should be used to display blocks.
-	 *
-	 * @return boolean
-	 */
-	public function is_block_templates_controller_refactor_enabled() {
```

Comment:
This was not being used, it had just been forgotten to remove it when this was refactored a while ago.